Modules/BenchPress.Azure/Public/Confirm-ApiManagementApi.ps1

```diff
@@ -0,0 +1,56 @@
+# INLINE_SKIP
+using module ./../Classes/ConfirmResult.psm1
+
+. $PSScriptRoot/../Private/Connect-Account.ps1
+# end INLINE_SKIP
+
+function Confirm-ApiManagementApi {
+  <#
+    .SYNOPSIS
+      Confirms that an API Management API exists.
+
+    .DESCRIPTION
+      The Confirm-AzBPApiManagementApi cmdlet gets an API Management API using the specified API, API Management
+      Service, and Resource Group names.
+
+    .PARAMETER ResourceGroupName
+      Specifies the name of the resource group under which an API Management service is deployed.
+
+    .PARAMETER ServiceName
+      Specifies the name of the deployed API Management service.
+
+    .PARAMETER Name
+      Specifies the name of the API to get.
+
+    .EXAMPLE
+      Confirm-AzBPApiManagementApi -ResourceGroupName "rgbenchpresstest" -ServiceName "servicetest" -Name "benchpresstest"
+
+    .INPUTS
+      System.String
+
+    .OUTPUTS
+      ConfirmResult
+  #>
+  [CmdletBinding()]
+  [OutputType([ConfirmResult])]
+  param (
+    [Parameter(Mandatory=$true)]
+    [string]$ResourceGroupName,
+
+    [Parameter(Mandatory=$true)]
+    [string]$ServiceName,
+
+    [Parameter(Mandatory=$true)]
+    [string]$Name
+  )
+  Begin {
+    $ConnectResults = Connect-Account
+  }
+  Process {
+    $Resource = New-AzApiManagementContext -ResourceGroupName $ResourceGroupName -ServiceName $ServiceName
```

Comment:
I don't see the comment Caroline mentioned (unless this is an old commit)

Reply:
@quigg-caroline 

Just to close the loop on this... I initially misunderstood Caroline's comment. Because it was brought up by Mary that `Confirm-ApiManagmentLogger` does not use the pipeline compared to the other `Confirm-ApiManagement*` files my mind immediately jumped to that file to document the difference. That's what "Done" meant in response to Caroline's comment, misunderstanding the comment.

Now that I re-read the comment, I realize that is not what was being asked. Regarding writing a comment for pipelining: I don't feel that this is necessary because it is the "PowerShell" way of doing things. Pipelining is the expected, and understood, way of writing PowerShell code and a comment explaining it would be really odd.